stratum/hal/lib/barefoot/bfrt_table_manager_test.cc

```diff
@@ -51,84 +51,53 @@ class BfrtTableManagerTest : public ::testing::Test {
     const std::string kSamplePipelineText = R"pb(
       programs {
         name: "test pipeline config",
-        p4info {
-          pkg_info {
-            arch: "tna"
-          }
-          tables {
-            preamble {
-              id: 33583783
-              name: "Ingress.control.table1"
-            }
-            match_fields {
-              id: 1
-              name: "field1"
-              bitwidth: 9
-              match_type: EXACT
-            }
-            match_fields {
-              id: 2
-              name: "field2"
-              bitwidth: 12
-              match_type: TERNARY
-            }
-            match_fields {
-              id: 3
-              name: "field3"
-              bitwidth: 15
-              match_type: RANGE
-            }
-            action_refs {
-              id: 16794911
-            }
-            const_default_action_id: 16836487
-            direct_resource_ids: 318814845
-            size: 1024
-          }
-          actions {
-            preamble {
-              id: 16794911
-              name: "Ingress.control.action1"
-            }
-            params {
-              id: 1
-              name: "vlan_id"
-              bitwidth: 12
-            }
-          }
-          direct_counters {
-            preamble {
-              id: 318814845
-              name: "Ingress.control.counter1"
-            }
-            spec {
-              unit: BOTH
-            }
-            direct_table_id: 33583783
-          }
-          meters {
-            preamble {
-              id: 55555
-              name: "Ingress.control.meter_bytes"
-              alias: "meter_bytes"
-            }
-            spec {
-              unit: BYTES
-            }
-            size: 500
-          }
-          meters {
-            preamble {
-              id: 55556
-              name: "Ingress.control.meter_packets"
-              alias: "meter_packets"
-            }
-            spec {
-              unit: PACKETS
-            }
-            size: 500
-          }
+        p4info {pkg_info {arch: "tna"} tables {
+          preamble {id: 33583783 name: "Ingress.control.table1"}
+          match_fields {id: 1 name: "field1" bitwidth: 9 match_type: EXACT}
+          match_fields {id: 2 name: "field2" bitwidth: 12 match_type: TERNARY}
+          match_fields {id: 3 name: "field3" bitwidth: 15 match_type: RANGE}
+          action_refs {id: 16794911}
+          const_default_action_id: 16836487
+          direct_resource_ids: 318814845
+          size: 1024
```

Comment:
Can you revert the textproto formatting?

clang-format should have not picked that up, as it's disabled in the config: https://github.com/stratum/stratum/blob/cb10d8501bd13b66b4c41f8a3d8139747d228765/.clang-format#L8-L10

Having to do it manually is unfortunate, but the autoformat produces suboptimal results.

Reply:
Sure, no problem.